docs/src/main/paradox/cluster-http-management.md

```diff
@@ -146,3 +146,23 @@ Example response:
          }
        ]
      }
+
+## Hosting the routes in an existing Akka HTTP server
+
+Starting `AkkaMangement` starts an Akka HTTP server and hosts the Cluster HTTP Routes. The routes instead can be added
```

Comment:
perhaps 'The routes can also be added..' rather than 'instead'?

Reply:
:+1: